src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTableIntegrationTest.java

```diff
@@ -236,4 +237,44 @@ public void testMoltenOutputWithMultipleAlleles() throws IOException {
         spec.setTrimWhiteSpace(false);
         spec.executeTest("testMoltenOutputWithMultipleAlleles", this);
     }
+
+    @Test
+    public void testNoFieldsSpecified() throws IOException {
```

Comment:
This test case should probably be renamed `testNoFieldsSpecifiedNoSamples()` for clarity (and rename the corresponding expected output file)